pkg/hubble/exporter/metrics.go

```diff
@@ -0,0 +1,104 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package exporter
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/cilium/cilium/pkg/hubble/metrics"
+	"github.com/cilium/cilium/pkg/hubble/metrics/api"
+	"github.com/cilium/cilium/pkg/time"
+)
+
+var (
```

Comment:
Can you document these metrics in `Documentation/observability/metrics.rst`?

Reply:
Added